src/fortran/.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file is not required.  A .gitignore file should be place in the root directory of a repository in any case.

Reply:
Removed .gitignore in src/fortran.